crates/trigger/Cargo.toml

```diff
@@ -32,4 +32,4 @@ wasmtime = { workspace = true }
 [dev-dependencies]
 tempfile = "3.3.0"
 toml = "0.5"
-tokio = { version = "1.0", features = ["rt", "macros"] }
```

Comment:
This is the one change I'm a bit unsure about from this PR. Did we have a reason for pinning tokio here at `1.0`?

Reply:
cc @lann @radu-matei